src/TwigComponent/doc/index.rst

```diff
@@ -365,6 +365,19 @@ You can even give the block default content. See
 :ref:`Passing HTML to Components via Block <embedded-components>`
 for more info.
 
+The only limitation when defining contents inside a component using the HTML syntax
+is that you cannot import macros using the ``_self`` keyword. You must always use
+the full template path:
+
+    <twig:Alert>
+        {# ❌ this won't work #}
+        {% from _self import message_formatter %}
+        {# ✅ this works as expected #}
+        {% from 'some/path/template.html.twig' import message_formatter %}
+
+        {{ message_formatter('...') }}
+    </twig:Alert>
```

Comment:
Not a RST expert here, but don't you need to use `.. code-block:: html+twig` to correctly highlight this code block?

Reply:
Hugo, you are absolutely right. I just fixed it. Thanks.